OC4IDS_Publisher_Status_Report.ipynb

```diff
@@ -0,0 +1,1796 @@
+{
```

Comment:
<p>This and the following other date cells, in the returned table the column with the date is titled "count" which is confusing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-contracting/notebooks-oc4ids/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>Updated column names to be the same as the name of the check. </p>